docs/articles/documentation/recipes/integrating-testcafe-with-ci-systems/azure-devops.md

```diff
@@ -0,0 +1,204 @@
+
+## Step 1 - Add TestCafe Dependency to the Project
+
+Open the repository's root directory and execute the following command:
+
+```sh
+npm --save-dev testcafe
+```
+
+This command installs the latest TestCafe version locally and adds it to the `devDependencies` section in the `package.json` file.
+
+```json
+{
+    "devDependencies": {
+        "testcafe": "*"
+    }
+}
+```
+
+## Step 2 - Provide a Command to Run Tests
+
+Since TestCafe is installed locally, a test run command that launches TestCafe should be also added to `package.json` to the `scripts` section.
+
+```json
+{
+    "scripts": {
+        "test": "testcafe chrome:headless tests/**/*"
+    },
+
+    "devDependencies": {
+        "testcafe": "*"
+    }
+}
+```
+
+This allows you to run tests using `npm test`.
+
+## Step 3 - Configure a Build Task
+
+Now you are ready to create a build task.
+
+### Configure Build Task on Azure Pipelines
+
+Open `azure-pipelines.yml` or create it in the repository root and add the following [job](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/phases?view=vsts&tabs=yaml).
+
+```yaml
+jobs:
+- job: e2e_tests
+  pool:
+    vmImage: 'Ubuntu 16.04'
+  steps:
+  - task: NodeTool@0
+    inputs:
+      # Replace '10.14' with the latest Node.js LTS version
+      versionSpec: '10.14'
+    displayName: 'Install Node.js'
+  - script: npm install
+    displayName: 'Install TestCafe'
+  - script: npm test
+    displayName: 'Run TestCafe Tests'
+```
+
+* `jobs` - the list of [jobs](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/phases?view=vsts&tabs=yaml).
+* `job` - the job name. You can choose any name you wish.
+* `pool` - the [agent pool](https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/pools-queues?view=vsts) that executes this build job.
+* `vmImage` - the [agent's](https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/agents?view=vsts) virtual machine image name. This tutorial uses a [Microsoft-hosted agent](https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=vsts&tabs=yaml) that runs on an Ubuntu 16.04 machine. You can find the list of software installed on this machine in the [Azure Pipelines GitHub repository](https://github.com/Microsoft/azure-pipelines-image-generation/blob/master/images/linux/Ubuntu1604-README.md).
+* `steps` - the list of steps performed when executing a job.
+* `task` - adds a [Node.js installer task](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/tool/node-js?view=vsts)  to the job. This task installs Node.js and adds it to the `PATH` variable.
+* `inputs` - specifies input variables for the task.
+* `versionSpec` - the Node.js version to install. You can find the latest LTS version number on the [Node.js website](https://nodejs.org/en/).
```

Comment:
You can find the latest LTS version number on -> The latest LTS version number is available on